go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/go-gl/example
 
-go 1.14
+go 1.22.0
```

Comment:
Yes, embed is available starting with 1.16. Updating to 1.22.0 is a part of "modernizing" the example.

There are several reasons not to try to stick with Go 1.16. By now that's an old, unsupported version. It doesn't get bug fixes or security vulnerability fixes. If someone reports a problem in the issue tracker of this repo and they say they're using 1.16, that creates busy work and latency - first we need to ask the person to upgrade to a newer, supported version of Go and see if the problem still happens, and then we can start trying to investigate the problem.

Furthermore, it may or may not work in CI (e.g., see [here](https://github.com/go-gl/example/actions/runs/11655304721)). I don't see a good reason to try to spend time on it, given 1.23 and 1.22 are available by now. They're fully backwards compatible and will work on existing Go code like an older 1.16 toolchain, just with better performance, fewer bugs, better support for current systems.

Reply:
I don't have a strong opinion and I'd use the oldest version whenever possible, but I understand your thoughts. Thanks!

> There are several reasons not to try to stick with Go 1.16. By now that's an old, unsupported version. It doesn't get bug fixes or security vulnerability fixes.

For example, Ubuntu's package manager provides outdated version of Go, but with security fixes by the Ubuntu package manager team. gccgo also supports Go 1.18 and not Go 1.19 and older. It is not feasible to support all the such old versions, but there are some situations to use old Go in the real world. Thus, I thought there is no strong reason to drop the old version even when we could support it.

> Furthermore, it may or may not work in CI (e.g., see [here](https://github.com/go-gl/example/actions/runs/11655304721)).

I think this is due to the last `.0` in `1.22.0` which was not expected by old Go.